server/services/participant-status.js

```diff
@@ -143,7 +147,7 @@ const setParticipantStatus = async (
       id: participantId,
     });
     // Now check if current status is archived then set interested flag
-    if (status === ARCHIVED) {
+    if (status === ARCHIVED && !isRosComplete) {
```

Comment:
In the user story, I don't see any criteria for not withdrawing `participant` this can impact app behavior. Please discuss this with me.

Reply:
Before I had this in here, the participant was not removed from the ROS list, it just marked them as Withdrawn when I attempted to archive them. I don't think it makes sense to withdraw them- if I graduate from a program, I don't withdraw from it, I complete it. Maybe we can do something else to mark them as "completed" instead.